test/bigml/test/api/t01.clj

```diff
@@ -0,0 +1,75 @@
+;; Copyright 2012, 2016 BigML
+;; Licensed under the Apache License, Version 2.0
+;; http://www.apache.org/licenses/LICENSE-2.0
+
+(ns bigml.test.api.t01
+  "Contains a few examples that exercise BigML's API"
+  (:require (bigml.api [core :as api]
+                       [source :as source]
+                       [dataset :as dataset]
+                       [model :as model]
+                       [prediction :as prediction]
+                       [evaluation :as evaluation]
+                       [cluster :as cluster]
+                       [centroid :as centroid])
+            (bigml.test.api [utils :as test])
+            (clojure.data [csv :as csv])
+            (clojure.java [io :as io]))
+  (:use clojure.test))
+
+(defn take-csv
+  "Takes file name and reads data."
+  [fname]
+  (with-open [file (io/reader fname)]
+    (doall (csv/read-csv (slurp file)))))
+
+(defn create-get-cleanup
+  "This function takes a sequence of resource to create (e.g.,
+   [source dataset model] and a prediction to check, which is
+   an array whose first item is the inputs to use for the prediction
+   and the second the expected outcome."
+  [res-type test]
```

Comment:
Little tip - you could use bindings here to reference the first two elements of `test` and skip `first` and `second`:

```clojure
(defn create-get-cleanup
   [res-type [uuid params]]
   (test/create-get-cleanup res-type uuid params))
```